### PR TITLE
sessionctx/binloginfo: write DDL binlog in the same way as DML binlog.

### PR DESCRIPTION
**Pull request by coocood**
Provide correct startTS and commitTS, and write rollback ddl binlog if transaction rollback.

### CODE REVIEW

**Comment by coocood** [comment]
@iamxy @GregoryIan @shenli PTAL

File: store/tikv/txn.go [hunk 1]
```diff
@@ -138,10 +138,10 @@ func (txn *tikvTxn) Commit() error {
 	}
 	err = committer.Commit()
 	if err != nil {
-		committer.writeFinisheBinlog(binlog.BinlogType_Rollback, 0)
```
Comment (GregoryIan) [inline store/tikv/txn.go]: don't we have a postDDL type? If so, why just leave three types `BinlogType_Prewrite, BinlogType_Rollback, BinlogType_Commit`

Comment (tiancaiamao) [inline store/tikv/txn.go]: postDDL may be Commit or Rollback

**Comment by tiancaiamao** [comment]
LGTM

**Comment by GregoryIan** [comment]
LGTM

**Pull request merged**